tools/ovc/unit_tests/moc_tf_fe/conversion_basic_models_test.py

```diff
@@ -169,20 +60,21 @@ def test_cutting_fp32(self, input_freezing_value, inputs, expected,
 
```

Comment:
Now it seems no tests remains here

Reply:
It is planned to return input/output cut functionality when "cut" helper is created. So I left the tests here commented, so we could easily return them later.
But I can also remove this file to make code cleaner.